src/patient-lab-root.component.tsx

```diff
@@ -12,17 +13,22 @@ const swrConfiguration = {
 
 const Root: React.FC = () => {
   return (
-    <main>
-      <SWRConfig value={swrConfiguration}>
-        <BrowserRouter basename={spaRoot}>
-          <Route
-            exact
-            path={patientLabDetailsRoute}
-            component={PatientLabDetails}
-          />
-        </BrowserRouter>
-      </SWRConfig>
-    </main>
+    <UserHasAccess
+      privilege={privilegeLabLite}
+      unauthorisedResponse="You are unauthorised to view this page. Please contact your administrator"
```

Comment:
Two issues
1) Not clear on how do we plan to use this message?
2) Put this into the translation...

The goal should be that the consumer of the component `UserHasAccess` should have control on what to do when the user does not have access. Valid options are a) do nothing [current impl] b) redirect to some page c) render some component (which could also be a redirect)

Take a look at this [code](https://codesandbox.io/s/react-typescript-forked-2bsqmi?file=/src/App.tsx) for a sample implementation of what I meant...

Reply:
We have updated UserHasAccess in [openmrs-esm-core](https://github.com/openmrs/openmrs-esm-core/pull/472). Currently, we could pass a component as unauthorised response instead of string. 

Also, we are not doing redirects to login page at the moment.